components/breadcrumbs/breadcrumbs.js

```diff
@@ -0,0 +1,88 @@
+import './breadcrumb.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+/**
+ * A component to help users understand where they are within the LMS. would be used the parent of d2l-breadcrumb and will render them in a slot
+ */
+class BreadCrumbs extends RtlMixin(LitElement) {
```

Comment:
Class should just be `Breadcrumbs` to match casing.

Reply:
Done ✔️